sys/posix/inet/Kconfig

```
@@ -0,0 +1,12 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_POSIX_INET
+    bool "Posix INet"
```

```suggestion
    bool "Posix Inet"
```
?

---

I would actually prefer either all lower case or all caps. Those are the two ways they are used in the [POSIX specification](https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/basedefs/arpa_inet.h.html#tag_13_03) (apart from just spelling out Internet).

---

Or to be more precise
```suggestion
    bool "POSIX Internet Support"
```
?

---

Done